content/applications/productivity/iot/devices/scale.rst

```diff
@@ -31,24 +32,26 @@ so, go to the *IoT Box Home Page* and click on *drivers list*. Then, click on lo
 Use a Scale in Point of Sale
 ============================
 
-To use the scale in your *Point of Sale* app, go to :menuselection:`Point of Sale --> Configuration
---> Point of Sale`, open the one you want to configure, then click on *Edit* and enable the *IoT
-Box* feature.
+To use the scale in the :menuselection:`Point of Sale` app, go to :menuselection:`Point of Sale -->
+Configuration --> Point of Sale`, open the :menuselection:`POS` that the scale will be configured
+on, and enable the :guilabel:`IoT Box` feature (under :menuselection:`Connected Devices`).
 
 .. image:: scale/iot-box-pos.png
    :align: center
    :alt: View of the IoT box feature inside of the PoS settings.
 
-Now, choose the *IoT Box* in the dropdown menu and check the *Electronic Scale* option. Then, you
-hit save.
+Now, choose the **IoT Box** in the dropdown menu and check the :guilabel:`Electronic Scale` option.
+Then, click on save.
 
 .. image:: scale/electronic-scale-feature.png
    :align: center
    :alt: List of the external tools that can be used with PoS and the IoT box.
 
-The scale is now available in all your *PoS* sessions. Then, if a product has a price per weight
-set, clicking on it on the *PoS* screen opens the scale screen, where the cashier can weigh the
-product and add the correct price to the cart.
+.. Needs new screenshot.
```

Comment:
Still needs one or already added?

Reply:
It does still need a new screenshot, I'll get this added on.